src/reflect/scala/reflect/internal/Symbols.scala

```diff
@@ -756,19 +774,25 @@ trait Symbols extends api.Symbols { self: SymbolTable =>
     /** Does symbol have ALL the flags in `mask` set? */
     final def hasAllFlags(mask: Long): Boolean = getFlag(mask) == mask
 
-    def setFlag(mask: Long): this.type   = { _rawflags |= mask ; this }
-    def resetFlag(mask: Long): this.type = { _rawflags &= ~mask ; this }
-    def resetFlags(): Unit               = { rawflags = 0 }
+    def setFlag(mask: Long): this.type = {
```

Comment:
Since these operations are side-effectful, could their return types be changed to `Unit`?

Reply:
These are designed for chaining/fluent-style, hence `this.type` return type.

I pushed a much smaller diff without the inadvertant reformatting.